### ISSUE TITLE
Google Play Game Service Leaderboard Limit

### ISSUE THREAD

**Issue by BadSnowBall** [playgameservices/play-games-plugin-for-unity#2535]
We are asking a support that if there is any method to increase the limitation of leaderboards. Which means that if we have more than 70 songs, we will need over 70 several leaderboards to support this feature. We are looking for an official method to break this limitation instead of using a third part game service.

We are a small game Development Team. We published a rhythm game, Dot n Beat(com.dotnbeat.dancing.music.rhythm.handspeed). In this game, we are using the google play game service to support our leaderboard feature. And it seems like a popular feature in our game. However, we mention that there is a limitation on the max number of the leaderboards, which is 70 now. In our game, every song has a unique leaderboard. We are asking a reasonable support that if there is any method to increase this limitation. Which means that if we have more than 70 songs, we will need over 70 several leaderboards to support this feature. We are looking for an official method to break this limitation instead of using a third part game service.

**Comment by olehkuznetsov** [comment]
Do you use a single leaderboard UI only or all leaderboard UI as well?

**Comment by BadSnowBall** [comment]
Yes, we are using both. Every single song has a single leaderboard and there is a all leaderboard UI in our setting page.

**Comment by olehkuznetsov** [comment]
We have it in our plans, but this will not be a quick fix.

**Comment by xixigeek** [comment]
We need that too，pls update soon ☺

**Comment by hell425** [comment]
> We have it in our plans, but this will not be a quick fix.

I would like to know, is there a temporary alternative to this issue? Can we only access third-party plug-ins?

**Comment by alanlawrance** [comment]
Found this while searching for info on the hard limit of 70 leaderboards.   Is this still in the plans, and it there any timeframe?

Game Center on iOS allows for up to 500 leaderboards across up to 100 sets (with each set having a maximum of 100 leaderboards).

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
